installation/single_machine_docker.rst

```diff
@@ -5,24 +5,44 @@ Docker
 
 This section describes setting up a Citus cluster on a single machine using docker-compose.
 
-**1. Install docker and docker-compose**
+1. Install Docker Engine and docker-compose
+-------------------------------------------
 
-Follow the installation instructions for `Docker Engine <https://docs.docker.com/engine/installation/>`_ and `Docker Compose <https://docs.docker.com/compose/install/>`_ on your platform.
+Mac / Windows
+~~~~~~~~~~~~~
 
-**2. Start the Citus Cluster**
+* Install the `Docker Platform <https://www.docker.com/products/overview#/install_the_platform>`_.
+* Start Docker by clicking on the application's icon.
+
+Linux
+~~~~~
+
+.. code-block:: bash
+
+  curl -sSL https://get.docker.com/ | sudo bash
+  sudo usermod -aG docker $USER && exec su -l $USER
```

Comment:
I spinned up a Redhat instance on AWS and got stock at the ``exec su -l $USER`` step.

This is because I have ``sudo`` on AWS instances but not ``su``.